web/src/pages/UploadBooksPage.css

```diff
@@ -0,0 +1,207 @@
+p {
+  margin: 0px;
+}
+
+.title {
+  text-align: left;
+  margin-right: 40px;
+}
+
+.clickableText {
+  text-decoration: underline;
+  border: none;
+  outline: none;
+  background-color: var(--white);
+  cursor: pointer;
+}
+
+.row {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+.books {
+  width: 100%;
+  margin-top: 56px;
+  margin-bottom: 139px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+  grid-template-rows: 1fr;
+  grid-gap: 62px;
+}
+
+.searchContainer {
+  display: flex;
+  margin-top: 90px;
+  margin-bottom: 9px;
+}
+
+.search {
+  height: 70px;
+  width: 500px;
+  padding-left: 20px;
+  border: 1px solid var(--orange);
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+}
+
+.searchButton {
+  height: 70px;
+  width: 140px;
+  background-color: var(--orange);
+  border: none;
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+  cursor: pointer;
+}
+
+input[type=search] {
+  font-size: 28px;
```

Comment:
Not sure if we can use this, b/c the styles permeate through the entire app, not just page the css is stored in. It would apply the same style to other input boxes. Not sure if it matters tho, I would ask David

Reply:
Yeah, good catch, just tested and this is indeed globally-scoped.

Maybe we can look into [CSS modules](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet)?